src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -236,6 +236,23 @@ appears to the reader as the initial idea but works with doc tests:
 /// ```
 ```
 
+As of version 1.34.0, one can also omit the `fn main()`, but you will have to
+disambiguate the error type:
+
+```ignore
+/// ```
+/// use std::io;
+/// let mut input = String::new();
+/// io::stdin().read_line(&mut input)?;
+/// # Ok::<(), io:Error>(())
```

Comment:
Typo: `io:Error` should be `io::Error` :)

Reply:
Good catch. WIll fix in a follow-up.